client/packages/electron/src/electron.ts

```diff
@@ -225,3 +202,55 @@ process.on('uncaughtException', error => {
         at process.processTimers (node:internal/timers:502:7)
   */
 });
+
+// App data store
+type StoreType = {
+  [key: string]: string | null;
+};
+const store = new ElectronStore<StoreType>();
+// /Users/andreievg/Library/Application Support/open mSupply
+// make CLI for generating certs
+app.addListener(
+  'certificate-error',
+  (event, _webContents, url, _error, certificate, callback) => {
+    // Ignore SSL checks in debug mode
+    if (getDebugHost()) {
+      event.preventDefault();
+      return callback(true);
+    }
+
+    // Default behaviour if not connected to a server or if url is not connectedServer
+
+    if (!connectedServer) return callback(false);
+
+    if (!url.startsWith(frontEndHostUrl(connectedServer))) {
+      return callback(false);
+    }
+
+    // Match SSL fingerprint for server stored in app data
+
+    // Match by hardware id and port
+    let identifier = `${connectedServer.hardwareId}-${connectedServer.port}`;
```

Comment:
same as for Android, why use hardwareId and port and not just the fingerprint?

Reply:
I added my comment about reason to store hardwareId and port vs domain and port, 3b991d6d5c25fff0878ae69b2db3e706ae07ae18.

Didn't add the use case though, it's described in previous comment.